multiview/embed/pls.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------
+# Implements the Partial Least Squares view construction method
+# by wrapping the PLS regression function from sklearn. View
+# construction generates a new projection of the data.
+
+from sklearn.cross_decomposition import PLSRegression
+
+
+def partial_least_squares_embedding(
+    X, Y, n_components=2, return_embedding=True
```

Comment:
I think that returning embeddings is more useful (which is why I think you set the default to True?). So can we do this instead:
Have a parameter return_weights set to False. If they set to true, weights instead.

Lemme know what you think.